### PR TITLE
Release/2.0

### PR DESCRIPTION
**Pull request by nepalez**
Remove the reference to the policy from `Tram::Policy::Errors` collection.

### CODE REVIEW

File: lib/tram/policy/error.rb [hunk 1]
```diff
@@ -27,16 +27,20 @@ def self.new(value, **opts)
...
+    # @return [Array<String>] scope for error message translation
+    attr_reader :scope
+
+    # List of arguments for [I18n.t]
     #
     # @return [Array]
     #
     def item
-      [key, tags]
+      [key, tags.merge(scope: scope)]
```
Comment (sclinede) [inline lib/tram/policy/error.rb]:
```suggestion
      [key, {scope: scope}.merge(tags)]
```

Comment (sclinede) [reply lib/tram/policy/error.rb]: Otherwise, if we use `errors.merge` scope of merged errors would be incorrect)

Comment (nepalez) [reply lib/tram/policy/error.rb]: @sclinede I rewrote the way we store scopes of the error and dropped the separate argument for scopes. Now they are just a part of the `tags`, so this merging is not necessary.

File: lib/tram/policy/error.rb [hunk 2]
```diff
@@ -99,10 +95,10 @@ def contain?(some_key = nil, **some_tags)
...
     DEFAULT_SCOPE = %w[tram-policy errors].freeze
 
-    def initialize(key, scope, **tags)
-      @key   = key
-      @tags  = tags
-      @scope = scope || DEFAULT_SCOPE
+    def initialize(key, **tags)
+      @key  = key
+      @tags = tags
+      @tags[:scope] ||= DEFAULT_SCOPE if key.is_a?(Symbol)
```
Comment (sclinede) [inline lib/tram/policy/error.rb]: hmmm. I'm what do you think about having nil scope as a valid case?
I'm not sure about it, but...
maybe we should prefer 
```suggestion
      @tags[:scope] = @tags.fetch(:scope) { DEFAULT_SCOPE } if key.is_a?(Symbol)
```

Comment (nepalez) [reply lib/tram/policy/error.rb]: I couldn't imagine the case when this is necessary. But why not? At least, your version is less magical for explicit `nil`

**Comment by nepalez** [comment]
@sclinede wdyt? let's bump?

**Pull request merged**